processor/k8sattributesprocessor/config.go

```diff
@@ -38,6 +39,13 @@ type Config struct {
 	// Exclude section allows to define names of pod that should be
 	// ignored while tagging.
 	Exclude ExcludeConfig `mapstructure:"exclude"`
+
```

Comment:
If add new fields to config, the `README.md` should also be updated.

Reply:
done